web-client/src/presenter/actions/getDocumentQCInboxForSectionAction.test.js

```diff
@@ -4,7 +4,7 @@ import { presenter } from '../presenter-mock';
 import { runAction } from 'cerebral/test';
 
 describe('getDocumentQCInboxForSectionAction', () => {
-  const mockWorkItems = [{ docketEnryId: 1 }, { docketEntryId: 2 }];
+  const mockWorkItems = [{ docketEntryId: 1 }, { docketEntryId: 2 }];
```

Comment:
This led me to search for `docketEnry`, and I found a few other hits. Could we clean those up, too? 🙏

Reply:
done.